### PR TITLE
Issue 4 cors

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

**Comment by Shalsh23** [comment]
@balhoff Are these changes in line with what we discussed the other day?

File: build.sbt [hunk 1]
```diff
@@ -38,6 +38,8 @@ libraryDependencies ++= {
     "com.softwaremill.sttp.tapir" %% "tapir-openapi-circe-yaml" % tapirVersion,
     "com.softwaremill.sttp.tapir" %% "tapir-swagger-ui-http4s"  % tapirVersion,
     "org.http4s"                  %% "http4s-blaze-server"      % http4sVersion,
+    "org.http4s"                  %% "http4s-dsl"               % http4sVersion,
+    "org.http4s"                  %% "http4s-server"            % http4sVersion,
```
Comment (balhoff) [inline build.sbt:42]: Do you know if we need this if we have `http4s-blaze-server`?

File: src/main/scala/org/renci/cam/Server.scala [hunk 2]
```diff
@@ -53,7 +56,7 @@ object Server extends App {
         servr <-
           BlazeServerBuilder[Task](runtime.platform.executor.asEC)
             .bindHttp(appConfig.port, appConfig.host)
-            .withHttpApp(httpAppWithLogging)
+            .withHttpApp(CORS.httpApp(httpAppWithLogging))
```
Comment (balhoff) [inline src/main/scala/org/renci/cam/Server.scala:59]: I think here you can just use `CORS(httpAppWithLogging)`. Not sure that it matters at all.

File: build.sbt (hunk 1 above)
Comment (Shalsh23) [reply build.sbt:42]: Not sure about this, actually.

**Pull request merged**